src/ui/server/routes/insurance/your-buyer-details/index.ts

```diff
@@ -0,0 +1,11 @@
+// import express from 'express';
+// import { YOUR_BUYER } from '../../../constants/routes/insurance/your-buyer';
+// // import { ROUTES } from '../../../constants';
+// import { get as getaYourBuyerOrganisation } from '../../../controllers/insurance/your-buyer/buyer-oganisation';
+
+// // @ts-ignore
+// const yourBuyerRouter = express.Router();
+
+// yourBuyerRouter.get(YOUR_BUYER.YOUR_BUYER_DETAILS, getaYourBuyerOrganisation);
+
+// export default yourBuyerRouter;
```

Comment:
is there a reason these are commented out? And @ttbarnes should we follow the same thing for 'EXPORTER_BUSINESS' where routes are in 'YOUR_BUYER' file and then dereferenced in src/ui/server/routes/insurance/index.ts

Reply:
Yes exactly 😄 we follow the URL structure of each page, for every section - e.g these URLs defined in the design:

- exip.com/insurance/policy-and-exports/type-of-policy
- exip.com/insurance/your-business/company-details
- exip.com/insurance/your-buyer/buyer-company-organisation


Should be structured like this in the codebase:

```js
src/ui/server/routes/insurance/policy-and-exports/type-of-policy
src/ui/server/routes/insurance/your-business/company-details
src/ui/server/routes/insurance/your-buyer/buyer-company-organisation
```

Not like this:
```js
src/ui/server/routes/insurance/buyer-company-organisation
src/ui/server/routes/insurance/your-buyer-details
```
Because the "organisation" route has no context of what section it belongs to.

Same applies to all sepects e.g controllers, field definitions, e2e tests etc.

The template location of `your-buyer-details.njk`is perfect 👍